src/test/java/net/minecraftforge/debug/OredictItemPredicateTest.java

```diff
@@ -0,0 +1,14 @@
+package net.minecraftforge.debug;
+
+import net.minecraftforge.fml.common.Mod;
+
+/**
+ * Dummy mod class; the real work is in the advancements directory of this mod.
+ * This mod tests ore-dict advancement triggers.
+ * @author Landmaster
+ */
+@Mod(modid = OredictItemPredicateTest.MODID, name = "Oredict Item Predicate Test", version = "1.0", acceptableRemoteVersions = "*")
+public class OredictItemPredicateTest
+{
+    public static final String MODID = "oredict_predicate";
```

Comment:
@liach I would add an `ENABLED` flag here, except that I could not find a way to prevent the advancements from loading from Java. (The same thing happens with [this 1.12 test which is already in Forge proper.](https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/src/test/java/net/minecraftforge/debug/CraftingSystemTest.java)) Is there a hook anywhere in Forge to halt the loading of an advancement via an event?